ConnectionFactory.php

```diff
@@ -84,6 +84,9 @@ private function initializeTypes()
                 Type::addType($type, $typeConfig['class']);
             }
             if ($typeConfig['commented']) {
+                @trigger_error(
+                    'The commented option of the type configuration is deprecated since DoctrineBundle 1.7.
+                    Use the feature of the Type class Instead', E_USER_DEPRECATED);
```

Comment:
`instead` should not have a capital letter.

And you should not put a newline inside the string itself. If you want to use 2 lines in the source code, use concatenation.

Btw, I would add the `$type` in the message to make it easier to know which type should be updated.